client/src/pages/OfferHelp.js

```diff
@@ -169,9 +172,28 @@ const OfferHelp = withRouter((props) => {
     setState({ ...updatedAnswers });
     if (key === "location") {
       localStorage.setItem("offerHelpAnswers", JSON.stringify(updatedAnswers));
+      let query = {
+        objective: 1, // indexOf "Requesting Help"
+      };
+      if (updatedAnswers.location)
+        query.location = btoa(JSON.stringify(updatedAnswers.location));
+      const selectedFilters = { providers: [] };
+      if (updatedAnswers.providers) {
+        let organisationFilter = updatedAnswers.providers.filter(
+          (option) => option === "As an Organisation",
+        );
+        if (organisationFilter.length > 0) {
+          for (let i = 1; i < filters[1].options.length; ++i) {
+            selectedFilters.providers.push(filters[1].options[i].value);
+          }
+        } else {
+          selectedFilters.providers.push(filters[1].options[0].value);
```

Comment:
2. While selecting offer help-> volunteer/donor, only the post from the individuals are returned. See the filter applied in the GIF. It should return the post from both individual and Org.
```suggestion
              for (let i = 0; i < filters[1].options.length; ++i) {
            selectedFilters.providers.push(filters[1].options[i].value);
          }
```